test/e2e/framework/fixture.go

```diff
@@ -186,9 +195,15 @@ func NewWorkspaceFixture(t *testing.T, server RunningServer, orgClusterName stri
 	clusterClient, err := kcpclientset.NewClusterForConfig(cfg)
 	require.NoError(t, err, "failed to construct client for server")
 
+	labels := map[string]string{}
+	if !schedulable {
+		labels[nscontroller.WorkspaceSchedulableLabel] = "false"
```

Comment:
I find this odd. If you don't want scheduling, don't use Universal. What's the use-case?

Reply:
I was implicitly testing disabling scheduling for a universal workload. But sure, removed. I'll use a non-universal type instead.

Reply:
In this specific instance it's to disable scheduling so the workspace can be targeted by a syncer.